File: store/src/main/java/localfooddelivery/domain/DeliveryStarted.java
package localfooddelivery.domain;

import localfooddelivery.domain.*;
import localfooddelivery.infra.AbstractEvent;
import java.util.*;
import lombok.*;

@Data
@ToString
public class DeliveryStarted extends AbstractEvent {

    private Long id;
    private String menu;
    private String address;
    private String orderId;
    private String status;

    public DeliveryStarted(Order aggregate){
        super(aggregate);
    }
    public DeliveryStarted(){
        super();
    }
}

File: order/DeliveryStarted.py

from AbstractEvent import AbstractEvent
import json

class DeliveryStarted(AbstractEvent):
    id : int
    menu : str
    address : str
    orderId : str
    status : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.menu = None
        self.address = None
        self.orderId = None
        self.status = None

